coral/src/app/components/Modal.tsx

```diff
@@ -116,6 +116,7 @@ function Modal(props: ModalProps) {
           className={classes.modalWrapper}
         >
           <Box
+            aria-label={title}
```

Comment:
I added a follow up issue for that, to make sure all cases are covered correctly. I want to to check if it's better to be more flexible with aria-label (for now this works for, so I didn't want to pack it into this PR)